projects/plugins/jetpack/extensions/blocks/recipe/recipe-print.js

```diff
@@ -0,0 +1,87 @@
+const printBtns = document.querySelectorAll(
+	'.wp-block-jetpack-recipe-details__detail .wp-block-button__link'
+);
+const style = document.createElement( 'style' );
+
+printBtns.forEach( btn => {
+	btn.addEventListener( 'click', () => {
```

Comment:
Should we add a `e.preventDefault();` in here?

```suggestion
	btn.addEventListener(  'click', ( e ) => {
		e.preventDefault();
```

Reply:
For sure, couldn't hurt to ensure a rogue link doesn't override the button functionality. This has been handled!